reproject/scripts.py

```diff
@@ -0,0 +1,88 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import time
+from astropy import log
+from astropy.io import fits
+from . import reproject
+
+
+def reproject_cmd(infile, targetfile, outfile,
+                  inext=0, targetext=0, overwrite=False,
+                  method='bilinear', store_footprint=True):
+    """
+    Reproject one image onto another (file-based interface).
+
+    File-based interface to `reproject.reproject`.
+
+    Parameters
+    ----------
+    TODO
```

Comment:
I think we don't usually have these in the API docs and instead we should have a section in the docs mentioning the command-line tool and maybe just showing the output of ``--help``.